crates/ruff_python_ast/src/types.rs

```diff
@@ -76,3 +79,61 @@ impl<'a> From<&RefEquality<'a, Expr>> for &'a Expr {
 }
 
 pub type CallPath<'a> = smallvec::SmallVec<[&'a str; 8]>;
+
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct Import {
+    pub name: String,
```

Comment:
Have we looked ahead to what the needs will be for lint rules that will rely on this, like cyclic imports? It looks like this is a string representation of the module (e.g., `import foo` => `"foo"`). Will we need to translate it to a _file path_ at all? Do we have the information we need about the containing package?

Reply:
Very happy for others to chime in here, I'm most familiar with pylint as that's what I use on my day job, I ran through the list of items from #970, and honestly I think this is only useful for `cyclic-import R0401`.
One could also use the logic used to report the imports for `import-self (W0406)` I reckon, but that doesn't need knowledge of other imports.